src/main/java/mate/academy/spring/model/User.java

```diff
@@ -5,16 +5,17 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import mate.academy.spring.validation.Email;
 
 @Entity
 public class User {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @Column(unique = true)
+    @Email
```

Comment:
why did you write this annotation here? We should check parameters from dto

Reply:
Jakarta works.